server/huntflow_reloaded/scheduler.py

```diff
@@ -0,0 +1,58 @@
+""" Scheduler module """
+
+import re
+import json
+from datetime import timedelta, datetime
+
+from apscheduler.schedulers.tornado import TornadoScheduler
+
+_SCHEDULER = TornadoScheduler()
+
+def add(date, func, args=(), kwargs={}):  # pylint: disable=dangerous-default-value
+    """ Wrapper for adding new task to scheduler """
+    _SCHEDULER.add_job(
+        func,
+        'date',
+        run_date=date,
+        args=args,
+        kwargs=kwargs
+    )
+
+def create_event(ctx, message):
+    """
+    Create tasks in scheduler:
```

Comment:
Schedules the events:
* remind an hour in advance;
* remind in the morning of the event day;
* remind in the evening of the event day.